solidity/test/helpers/ContractRegistryClient.js

```diff
@@ -2,6 +2,7 @@ module.exports.CONTRACT_REGISTRY              = web3.fromAscii("ContractRegistry
 module.exports.BANCOR_NETWORK                 = web3.fromAscii("BancorNetwork");
 module.exports.BANCOR_FORMULA                 = web3.fromAscii("BancorFormula");
 module.exports.CONVERTER_FACTORY              = web3.fromAscii("ConverterFactory");
+module.exports.CONVERSION_PATH_FINDER         = web3.fromAscii("ConversionPathFinder");

```

Comment:
@yudilevi : Excellent choice for placing this line!

Reply:
I learned from the best